pkg/api/types.go

```diff
@@ -2427,7 +2427,7 @@ type SerializedReference struct {
 type EventSource struct {
 	// Component from which the event is generated.
 	Component string `json:"component,omitempty"`
-	// Host name on which the event is generated.
+	// Node name on which the event is generated.
```

Comment:
I did a search in the code base and it seems we never set the EventSource.Host. If that's true, we might be able to change the schema.

> We don't even publish the host name on a Node

Do you mean we can't find the host name from the Node api?